platform-core-module/src/test/java/com/softicar/platform/core/module/event/SystemEventBuilderTest.java

```diff
@@ -0,0 +1,18 @@
+package com.softicar.platform.core.module.event;
+
+import com.softicar.platform.core.module.test.AbstractCoreTest;
+import com.softicar.platform.db.core.transaction.DbTransaction;
+import org.junit.Test;
+
+public class SystemEventBuilderTest extends AbstractCoreTest {
+
+	@Test
+	public void testEventCreationInSeperateDBConnection() {
+
+		try (var t = new DbTransaction()) {
+			insertSystemInformationEvent("Test");
```

Comment:
We should probably assign the return value to a variable (of type `AGSystemEvent`) and assert sameness of references. Like so:
```
AGSystemEvent event;
try (var transaction = new DbTransaction()) {
	event = insertSystemInformationEvent("Test");
	transaction.rollback();
}

var events = AGSystemEvent.TABLE.loadAll();
assertEquals(1, events.size());
assertSame(event, events.get(0));

```